sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -465,33 +467,41 @@ private TimeZone getTimeZone() {
     }
   }
 
+  private @Nullable Float getBatteryTemperature(final @NotNull Intent batteryIntent) {
+    try {
+      int temperature = batteryIntent.getIntExtra(EXTRA_TEMPERATURE, -1);
```

Comment:
this won't work as a breadcrumb, I get too many events per minute and it can be very noisy, so temperature comes along with the event